tests/schema/hooks.test.ts

```diff
@@ -205,7 +201,7 @@ it('in production, throws if a wrong arg named is passed to ordering', async ()
     await generateSchemaAndTypes(datamodel, [Query, User])
   } catch (e) {
     expect(e).toMatchInlineSnapshot(
-      `[Error: Your GraphQL \`Query\` object definition is projecting a relational field \`users\`. On it, you are declaring that clients be able to order by Prisma \`User\` model field \`unknownField\`. However, your Prisma model \`User\` model has no such field \`unknownField\`]`,
+      `[Error: Could not find field 'Mutation.undefined' on type Mutation]`,
```

Comment:
It looks like proxies are broken

Reply:
Looks like a flubbed a merge conflict with a couple of your test changes. Resolved!